lib/repl.js

```diff
@@ -1262,7 +1262,7 @@ function defineDefaultCommands(repl) {
       const names = Object.keys(this.commands).sort();
       const longestNameLength = names.reduce((max, name) => {
         return Math.max(max, name.length);
```

Comment:
@silverwind `(max, name) => Math.max(max, name.length)` ? is it against style guide?